source/quick-start.txt

```diff
@@ -2,5 +2,175 @@
 Quick Start
 ===========
 
-
 .. default-domain:: mongodb
+
+Check for Node and NPM
+----------------------
+
+Ensure your system has Node greater than or equal to version 8 and
+NPM (Node Package Manager) installed. You can verify whether you have Node and
+NPM installed by executing the following in your terminal.
+
+First, check the version of Node
+
+.. code-block:: sh
+
+   node -v
+
+You should see output similar to the following:
+
+.. code-block:: sh
+
+   v12.14.1
+
+Next, check the version of NPM
+
+.. code-block:: sh
+
+   npm -v
+
+You should see output similar to the following:
+
+.. code-block:: sh
+
+   6.13.4
+
+If you do not see any output, you will need to install Node and NPM.
+Refer to `nodejs.org <https://nodejs.org>`_ for installation instructions.
+
+
+Initialize the Project
+----------------------
+
+The starting point for any Node application is initializing your project. Navigate
```

Comment:
I think we can omit this sentence; it's passive voice and does not add much value for Jasmine or Lucas.